packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/types/FlutterSignUpRequest.kt

```diff
@@ -74,17 +74,19 @@ data class FlutterSignUpRequest(val map: HashMap<String, *>) {
 
     companion object {
         fun validate(req : HashMap<String, *>?): Boolean {
-            var valid: Boolean = true;
             if (req == null) {
-                valid = false;
+                return false;
+            }
+            if (req.get("options") == null) {
```

Comment:
It looks like options is required. Should we not add the `@required` annotation in the API as well to make it apparent to customers, or pass a default value if possible.
eg.
```
Future<SignUpResult> signUp({@required String username, @required password, @required SignUpOptions options}
```
This is in line with suggestions 1 and 2 in #136